src/xdocs/index.xml.vm

```diff
@@ -96,6 +96,15 @@
         For the exact day, please follow our
         <a href="https://github.com/checkstyle/checkstyle/milestones">milestones</a>.
       </p>
+      <p>
+        Checkstyle is following Emotional Semantic Versioning:
+        <b>First</b>.<b>Second</b>.<b>Third</b>
+        <br/>
+        <b>First</b> digit means that some very noticable breaking compatibility happened.<br/>
```

Comment:
`digit means that some very noticable breaking compatibility happened or major conceptual change` - consider the 10.x.x bump due to ANTLR4 update, where we changed major core functionality but did not break anything (until we included other changes).

Reply:
I did not find good wording to express this, so decided to keep on breaking changes, as most usually it  always exists.

Reply:
done